ARCHITECTURE.md

```diff
@@ -4,23 +4,46 @@ This document describes the high-level architecture of PyMC3.
 If you want to familiarize yourself with the code base, you are just in the right place!
 
 
-## Bird's Eye View
+# Bird's Eye View
 
 ![Architecture](docs/Architecture.png)
 
 On the highest level, PyMC3 is a thing which lets you define probabilistic graphs or models
```

Comment:
Let's you define probabilistic graphs or models that can be easily used to compute log probabilities for posterior inference or to draw random samples for prior and posterior prediction.

Reply:
Done